tests/unit/splinter_shmem_test.c

```diff
@@ -578,37 +580,6 @@ CTEST2(splinter_shmem, test_realloc_of_free_fragments_uses_first_fit)
    platform_free(data->hid, keybuf_5MiB);
 }
 
-/*
- * ---------------------------------------------------------------------------
- * Test case to verify that configuration checks that shared segment size
- * is "big enough" to allocate memory for RC-allocator cache's lookup
- * array. For very large devices, with insufficiently sized shared memory
- * config, we will not be able to boot-up.
- * RESOLVE - This test case is still incomplete.
- * ---------------------------------------------------------------------------
- */
-CTEST2(splinter_shmem, test_large_dev_with_small_shmem_error_handling)
```

Comment:
Deleted this test case as it's too low-level detail. (And a bit time consuming to hammer out the logic to exercise this scenario. Not very important right now.)